sdks/python/setup.py

```diff
@@ -234,6 +234,10 @@ def get_version():
     'azure-core >=1.7.0',
 ]
 
+ML_REQUIREMENTS = [
+    'torch >= 1.10.2'
```

Comment:
I can't see the original change now, but I think this was in an extra. It would be a requirement that's added when you `pip install apache-beam[ml]`, so it wouldn't be a hard dependency for all users.

Regardless I think we should hold off adding a requirement spec anywhere, unless we need to communicate that there's restricted version range we support.

Reply:
For now, to allow most flexibility, we're going to support `pip install apache-beam pytorch`